### ISSUE TITLE
Undefined function or variable 'bn'.

### ISSUE THREAD

**Issue by wajihullahbaig** [zhanghang1989/ResNet-Matconvnet#1]
I get the "Undefined function or variable 'bn'." error when I run the command

> run_cifar_experiments([20 32 44 56 110], 'plain', 'gpus', [1]);

```
info = add_group(opts.networkType, net, n, info, 3, 16, 1, opts);
Undefined function or variable 'bn'.

Error in res_cifar_init>add_group (line 87)
  add_block_conv(net, sprintf('%d', info.lastIdx+1), lName, ...

Error in res_cifar_init (line 50)
info = add_group(opts.networkType, net, n, info, 3, 16, 1, opts);

Error in res_cifar (line 42)
    net = res_cifar_init(m, 'networkType', opts.modelType, ...

Error in run_cifar_experiments (line 30)
  [net,info] = res_cifar(Ns(i), 'modelType', MTs{i}, opts);

>> run_cifar_experiments([20 32 44 56 110], 'plain', 'gpus', [1]);
```

**Comment by zhanghang1989** [comment]
Hi,
Thank you very much for pointing out this bug! It is fixed in the commit [#8e3a4ab](https://github.com/zhanghang1989/ResNet-Matconvnet/commit/01d27901cc2dec9042bc77db7a7b0b3b75448eef)

It was my bad. When I was updating the 'resnet', I forgot to make related changes for 'plain' net. And obviously, the people who have used my code, only care about 'resnet' :)
It is good to know this bug. Please let me know how it goes on your machine.

Best,
Hang

**Comment by wajihullahbaig** [comment]
Thanks for the reply. I will be testing your code again in a day or two :).
